analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -80,11 +80,10 @@ class Bundler(
         ) = Bundler(managerName, analysisRoot, analyzerConfig, repoConfig)
     }
 
-    override fun command(workingDir: File?) = if (Os.isWindows) "bundle.bat" else "bundle"
+    override fun command(workingDir: File?) = if (Os.isWindows) "ruby.bat" else "ruby"
 
-    override fun transformVersion(output: String) = output.removePrefix("Bundler version ")
-
-    override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[1.16,)")
+    override fun transformVersion(output: String) = output.removePrefix("ruby ").substringBefore("p")
```

Comment:
This is what I needed to pass the version check on my machine, since I do not get Docker to run. Maybe there is a better solution.